### ISSUE TITLE
H5074 doesn't seem to be able to connect

### ISSUE THREAD

**Issue by crookedstorm** [asednev/homebridge-plugin-govee#64]
**Describe Your Problem:**

New H5074 tested to be working with the official app doesn't get picked up by this plugin. It seems to be complaining about vendor info. I'd be happy to figure out some hack with udev or whatever, but I'm pretty new to BLE. I did notice that the MAC vendor portion is different than what was in the issue that lead to initial support, but it is typical for recent Govee stuff, so nothing terribly strange except the "undefined" manufacturer data in debug mode?

**Logs:**

```
a4c138476b53: 88ec000908a80e6402
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -31
a4c138476b53: 88ec000908a80e6402
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -31
a4c138476b53: 88ec000908a80e6402
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -47
a4c138476b53: 88ec000908a80e6402
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -39
invalid peripheral, manufacturerData=[4c000215494e54454c4c495f524f434b535f48575075f2ffc2]
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -17
invalid peripheral, manufacturerData=[4c000215494e54454c4c495f524f434b535f48575075f2ffc2]
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -17
invalid peripheral, manufacturerData=[4c000215494e54454c4c495f524f434b535f48575075f2ffc2]
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -31
invalid peripheral, manufacturerData=[4c000215494e54454c4c495f524f434b535f48575075f2ffc2]
```

in debug mode earlier I was also getting:
```
invalid peripheral, manufacturerData=[undefined]
```
But that's not what I see now.

**Plugin Config:**

```json
{
    "name": "Govee Sensors",
    "batteryThreshold": 25,
    "debug": true,
    "humidityOffset": 0,
    "platform": "GoveeHomebridgePlugin"
}
```

**Screenshots:**
None right now. The logs are all that show up in debug mode.

**Environment:**

* **Plugin Version**: 1.1.16 (also tried 1.0.8 for science)
* **Homebridge Version**: 1.4.0
* **Node.js Version**: v16.14.2
* **NPM Version**: 8.5.0
* **Operating System**: Raspbian GNU/Linux 11 (bullseye) 

From the Govee app when I had it installed for testing:
* **Firmware Version**: 1.00.01
* **Hardware Version**: 2.00.01
* **Product Model**: H5074

**Comment by crookedstorm** [comment]
Tried popping the battery out of the sensor for a bit and I got back to:
```
discovered a4c138476b53 a4c138476b53 a4:c1:38:47:6b:53 disconnected -28
invalid peripheral, manufacturerData=[undefined]
```

This doesn't really seem like progress, but I thought I would report it if that's interesting.

**Comment by crookedstorm** [comment]
Ok, actually, I see the values now. It's working, but it is doing the disconnecting and possible stale values thing I see in #40, which I see closed as a possible dupe of #19. I'll close this for now pending resolution of that other thing. Thanks!
